docs/Makefile

```diff
@@ -14,7 +14,11 @@ ALLSPHINXOPTS   = -d $(BUILDDIR)/doctrees $(PAPEROPT_$(PAPER)) $(SPHINXOPTS) .
 # the i18n builder cannot share the environment and doctrees with the others
 I18NSPHINXOPTS  = $(PAPEROPT_$(PAPER)) $(SPHINXOPTS) .
 
-.PHONY: help clean html dirhtml singlehtml pickle json htmlhelp qthelp devhelp epub latex latexpdf text man changes linkcheck doctest gettext coverage
+GENKEYIMG_SCRIPT = ../scripts/gen-keybinding-img -o ./_static/keybindings
+GENKEYIMG_REMOVE_IN_RST = sed -i '/^\.\. LS_PNG/,/^\.\. END_LS_PNG/{/^\.\. image/d}' manual/commands/keybindings.rst
+GENKEYIMG_INSERT_IN_RST = sed -i '/^\.\. END_LS_PNG/e find _static/keybindings/* | awk '"'"'{ print length, $$0 }'"'"' | sort -n | cut -d" " -f2- | awk '"'"'{print ".. image:: /" $$1}'"'"'' manual/commands/keybindings.rst
```

Comment:
Why do we need these two?

Reply:
this updates docs to list current images automatically

Reply:
Are you saying that it adds these lines? https://github.com/qtile/qtile/pull/1631/files#diff-edea8d20248b32a41e85854cd1dc0defR10-R12

If so, why do we need to automate this? Can't we just keep the lines as is and update if/when needed?

Reply:
yes, it adds these image source lines.
You want to say that manual is better then automatically? :) Then why you are using tiling wm? :D
I think that manual intervention is place to forget.

Reply:
But these lines are never going to change, right? I'm also using a tiling WM because it's minimalist and doesn't do too much. :wink: